### PR TITLE
JavaScript group capstone: day 1

### PR DESCRIPTION
**Pull request by vicmaburrito**
Create API call and header HTML

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,11 +1,88 @@
...
+                                TV Shows
+                            </a>
+                            <div class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink">
+                                <a class="dropdown-item" href="#">Action</a>
+                                <a class="dropdown-item" href="#">Another action</a>
+                                <a class="dropdown-item" href="#">Something else here</a>
+                            </div>
+                        </li>
+                    </ul>
+                </div>
```
Comment (rtonata88) [inline src/index.html:61]: Some of the menu items here have dropdown lists, however none of them are responding. Can you kindly take a look at this bug.

**Review by rtonata88 — DISMISSED** [review]
Hello @vicmaburrito 
# STATUS: REQUIRED CHANGES ♻️ 
Job well done so far on the header of the main site 😄. Just one more thing that needs fixing before we merge into `dev` branch. Please see my comments below.

There are also some linter errors 🐛. Can you take a long at them

File: src/api.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+const baseUrl = "https://api.tvmaze.com";
+export const mainInformationUrl = function (id) {
```
Comment (victor-aldaraca) [inline src/api.js]: Please note the possibility of using arrow

File: src/involvement.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    method: "POST",
+    body: data,
+    headers: {
+      Accept: "application/json",
+      "Content-Type": "application/json",
+    },
+  });
+};
+
+export const scheduleTitlesCounter = (schedule) => {
```
Comment (victor-aldaraca) [inline src/involvement.js:14]: Note to move the returned value immediately after the `=>`
```js
export const scheduleTitlesCounter = (schedule) => schedule.length;
```

**Review by vicmaburrito — COMMENTED** [review]
# Great job so far!
Keep doing like this, you are just on the right path!
## Required Changes ♻️.......

**Pull request merged**